rosplan_planning_system/scripts/direct_esterel_parser.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
This is a new node in ROSPlan. Should it be a part of this PR? It looks as though it is used for testing, in which case it should either be removed, or moved to a separate PR in rosplan_demos.

Reply:
Yes, it should be part of the pull request. 
As CHIMP already has an internal representation in form of a temporal constraint network, I added an option to CHIMP such that it can directly write the plan's Esterel representation to a file.
This node just converts this output into a `EsterelPlan` ROS message and publishes it.

Reply:
That makes sense. Might be worth editing the comment on line 12 to specify that it means ESTEREL plan YAML as string. Planner output is normally something else.

Reply:
You are right. I have updated the comment.